SPECS/ocaml-parsexp/ocaml-parsexp.spec

```diff
@@ -0,0 +1,130 @@
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+%ifnarch %{ocaml_native_compiler}
+%global debug_package %{nil}
+%endif
+
+%global srcname parsexp
+
+Name:           ocaml-%{srcname}
+Version:        0.16.0
+Release:        1%{?dist}
+Summary:        S-expression parsing library
+
+License:        MIT
+URL:            https://github.com/janestreet/parsexp
+Source0:        %{url}/archive/v%{version}/%{srcname}-%{version}.tar.gz
+
+BuildRequires:  ocaml >= 5.1.1
+BuildRequires:  ocaml-dune >= 2.0.0
+BuildRequires:  ocaml-sexplib0-devel >= 0.16
+
+%description
+This library provides generic parsers for parsing S-expressions from
+strings or other media.
+
+The library is focused on performance but still provides full generic
+parsers that can be used effortlessly with strings, bigstrings, lexing
+buffers, character streams or any other source.
+
+It provides three different classes of parsers:
+- the normal parsers, producing [Sexp.t] or [Sexp.t list] values;
+- the parsers with positions, building compact position sequences so
+  that one can recover original positions in order to properly report
+  error locations at little cost; and
+- the Concrete Syntax Tree parsers, producing values of type
+  [Parsexp.Cst.t] which record the concrete layout of the s-expression
+  syntax, including comments.
+
+This library is portable and doesn't provide I/O functions.  To read
+s-expressions from files or other external sources, you should use
+parsexp_io.
+
+%package        devel
+Summary:        Development files for %{name}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+Requires:       ocaml-sexplib0-devel%{?_isa}
+
+%description    devel
+The %{name}-devel package contains libraries and signature files for
+developing applications that use %{name}.
+
+%prep
+%autosetup -n parsexp-%{version}
+
+%build
+%dune_build
+
+%install
+%dune_install
+
+%check
+%dune_check
+
+%files -f .ofiles
+%doc CHANGES.md README.org
+%license LICENSE.md
+
+%files devel -f .ofiles-devel
+
+%changelog
+* Thu May 02 2024 Mykhailo Bykhovtsev <mbykhovtsev@microsft.com> - 0.16.0-1
+- Converted spec file to match with Fedora 41.
+- Upgraded to version 0.16.0
+
+* Tue Jan 18 2022 Thomas Crain <thcrain@microsoft.com> - 0.15.0-1
+- Upgrade to latest version
+- License verified
+
+* Fri Oct 15 2021 Pawel Winogrodzki <pawelwi@microsoft.com> - 0.14.0-9
```

Comment:
I think you missed removing `SPECS/ocaml-parsexp`

Reply:
Removed this and other missed packages